src/OrchardCore.Modules/OrchardCore.ContentLocalization/Shortcodes/LocalizationShortcodeProvider.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Globalization;
+using System.Threading.Tasks;
+using Shortcodes;
+
+namespace OrchardCore.ContentLocalization.Shortcodes
+{
+    public class LocalizationShortcodeProvider : IShortcodeProvider
+    {
+        public const string ShortCodeIdentifier = "locale";
+        private static readonly ValueTask<string> Null = new ValueTask<string>((string)null);
+
+        public ValueTask<string> EvaluateAsync(string identifier, Arguments arguments, string content, Context context)
+        {
+            if (identifier != ShortCodeIdentifier)
+            {
+                return Null;
+            }
+
+            var language = arguments.NamedOrDefault("lang");
+            var currentCulture = CultureInfo.CurrentCulture.Name;
+            if(!string.Equals(language, currentCulture, StringComparison.InvariantCultureIgnoreCase))
```

Comment:
I think it should also fallback to parent cultures. So if the page is in en-US a [locale en] will also work

Reply:
Done